Content.Server/Doors/Electronics/Systems/DoorElectronicsSystem.cs

```diff
@@ -0,0 +1,84 @@
+using System.Linq;
+using Content.Shared.Access.Components;
+using Content.Shared.DeviceNetwork.Components;
+using Content.Shared.Doors;
+using Content.Shared.Doors.Electronics;
+using Content.Shared.Interaction;
+using Content.Server.Doors.Electronics;
+using Robust.Server.GameObjects;
+
+namespace Content.Server.Doors.Electronics;
+
+public sealed class DoorElectronicsSystem : EntitySystem
+{
+    [Dependency] private readonly UserInterfaceSystem _uiSystem = default!;
+
+    public override void Initialize()
+    {
+        base.Initialize();
+        SubscribeLocalEvent<DoorElectronicsComponent, DoorElectronicsUpdateConfigurationMessage>(OnChangeConfiguration);
+        SubscribeLocalEvent<DoorElectronicsComponent, DoorElectronicsRefreshUiMessage>(OnRefreshUi);
+        SubscribeLocalEvent<DoorElectronicsComponent, InteractUsingEvent>(OnInteractUsing);
+    }
+
+    public void UpdateUserInterface(EntityUid uid, DoorElectronicsComponent component)
+    {
+        List<string> accesses = new List<string>();
+
+        if (TryComp<AccessReaderComponent>(component.Owner, out var accessReader))
```

Comment:
.Owner is deprecated, pls if you copy code from somewhere else at least make sure there are no obsolete warnings